tutorials/scripting/gdextension/gdextension_c_example.rst

```diff
@@ -0,0 +1,2165 @@
+allocated for the custom data. It is not necessary to destruct the Godot object
+since that will be taken care of by the engine itself.
+
+A demo project
+--------------
+
+Now that we can create and free our custom object, we should be able to try it
+out in an actual project. For this, you need to open Godot and create a new
+project on the ``demo`` folder. The project manager may warn you the folder
+isn't empty if you have compiled the extension before, you can safely ignore
+this warning this time.
+
+If you didn't compile the extension yet, it is the time to do it now. To do
+that, open a terminal or command prompt, navigate to the root folder of the
+extension and run ``scons``. It should compile quickly since the extension is
+very simple.
+
+Then, create a file called ``gdexample.gdextension`` inside the ``demo`` folder.
+This is a Godot resource that describes the extension, allowing the engine to
+properly load it. Put the following content in this file:
+
+.. code-block::
+
+    [configuration]
+
+    entry_symbol = "gdexample_library_init"
+    compatibility_minimum = "4.2"
+
+    [libraries]
+    macos.debug = "res://bin/libgdexample.dylib"
+    linux.debug = "res://bin/libgdexample.so"
+    windows.debug = "res://bin/libgdexample.dll"
+
+As you can see, ``gdexample_library_init`` is the same name of the function we
+defined in our ``init.c`` file. It is important that the names match because it
+is how Godot calls the entry point of the extension.
+
+We also set the compatibility minimum to 4.2, since we are targeting this
+version. It should still work on later versions. If you are using a later Godot
+version and rely on the new features, you need to increase this value to a
+version number that has everything you use.
+
+In the ``[libraries]`` section we set up the paths to the shared library on
+different platforms. Here there's only the debug versions since that's what we
+are working on for the example. Using :ref:`Feature tags <doc_feature_tags>` you
+can fine tune this to also provide release versions, add more target OSes, as
+well as providing 32-bit and 64-bit binaries.
+
+You can also add library dependencies and custom icons for your classes in this
+file, but this is out of the scope for this tutorial.
+
+After saving the file, go back to the editor. Godot should automatically load
+the extension. Nothing will be seen because our extension only register a new
+class. To use such, class add a ``Node2D`` as a root of the scene. Move it to
+the middle of viewport for better visibility. Then add a new child node to the
+root and in the ``Create New Node`` dialog search for "GDExample", the name of
+our class, as it should be listed there. If it isn't, it means that Godot didn't
+load the extension properly, so try restarting the editor and retrace the steps
+to see if anything went missing.
+
+Our custom class is derived from ``Sprite2D``, so it has a ``texture`` property
```

Comment:
```suggestion
Our custom class is derived from ``Sprite2D``, so it has a **Texture** property
```
Inspector properties are capitalized as the appear in the inspector (with the capitalization prettifying enabled) and bold, because they are editor UI.

(It's kinda inconsistently applied but it's the current "standard")